--- a/umu.py
+++ b/umu.py
@@ -40,9 +40,9 @@ def main():
     # Ensure the script is not run as root
     if os.geteuid() == 0:
-        err: str = "This script should never be run as the root user"
-        log.error(err)
-        sys.exit(1)
+        # Bypassed for root-only embedded systems (ROCKNIX/JeOS)
+        pass
+        # err: str = "This script should never be run as the root user"
+        # log.error(err)
+        # sys.exit(1)
